pandas/tests/groupby/test_reductions.py

```diff
@@ -1057,3 +1057,27 @@ def test_regression_allowlist_methods(op, axis, skipna, sort):
         if sort:
             expected = expected.sort_index(axis=axis)
         tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("multiplier", range(90))
```

Comment:
I don't think we need multiple tests for this; okay to remove the parametrization.